app/views/companies/_search_form.html.haml

```diff
@@ -0,0 +1,49 @@
+= search_form_for @search_params, method: :get, url: companies_path,
+                                        class: 'form-horizontal' do |f|
+  .form-group
+    .col-sm-5
+      = f.label :business_categories_id_in,
+                "#{t('activerecord.models.business_category.one')}",
+                class: 'control-label',
+                style: 'text-align: left; font-size: 12px;'
+      = f.collection_select :business_categories_id_in,
```

Comment:
Is this happening in your environment?  Generally, categories are only added if there is a membership application associated with a category, and then a company gets associated with the category when an application (associated with the company) is accepted.

So, in production I would think that only categories associated with a company would appear in the selection list (they are appearing in the screenshots because I have manually added records to the DB without regard for the actual business rules and application workflow).